<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QMainWindow" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lytHomeBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lytHomeBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lytHomeBtn.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>lytHomeBtn.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="cmmndHomeBtn">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>4</y>
      <width>173</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#cmmndHomeBtn
{
border-image: url(:/cmmndHomeBtn.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sttsHomeBtn">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>4</y>
      <width>173</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#sttsHomeBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/sttsHomeBtn.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mbnHomeBtn">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>4</y>
      <width>174</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mbnHomeBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbnHomeBtn.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mbstHomeBtn">
    <property name="geometry">
     <rect>
      <x>613</x>
      <y>4</y>
      <width>172</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mbstHomeBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/mbstHomeBtn.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="lvlingHomeBtn">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>244</y>
      <width>173</width>
      <height>196</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>B Nazanin</family>
      <pointsize>19</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lvlingHomeBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lvlingHomeBtn.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clrHomeBtn">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>244</y>
      <width>173</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#clrHomeBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/clrHomeBtn.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tempHomeBtn">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>244</y>
      <width>175</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#tempHomeBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/tempHomeBtn.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pwrHomeBtn">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>244</y>
      <width>174</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pwrHomeBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/pwrHomeBtn.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tempLbl">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#tempLbl
{
color:rgb(56, 54, 55)
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;T=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="tmpValue">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#tmpValue
{
color: rgb(56, 54, 55);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;27 C &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="hmdLbl">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#hmdLbl
{
color:rgb(56, 54, 55)
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;H= &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="hmdValue">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#hmdValue
{
color:rgb(56, 54, 55)
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;18 %&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="kab1Lbl">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#kab1Lbl
{
color:rgb(56, 54, 55)
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;KABINE1:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="kab1stLbl">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0055ff;&quot;&gt;off&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="kab2Lbl">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#kab2Lbl
{
color:rgb(56, 54, 55)
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;KABINE2:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="kab2stLbl">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#00aa00;&quot;&gt;on&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="xlbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color : rgb(56, 54, 55)}</string>
    </property>
    <property name="text">
     <string>X = </string>
    </property>
   </widget>
   <widget class="QLabel" name="ylbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color : rgb(56, 54, 55)}</string>
    </property>
    <property name="text">
     <string>Y = </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
